packages/entity-browser/src/util/reduxForms.js

```diff
@@ -34,6 +47,12 @@ export const entityToFormValues = entity => {
   return result
 }
 
+export const getDirtyFields = (initialValues, values) => {
+  return _reduce(initialValues, function(result, value, key) {
+    return _isEqual(value, values[key]) ? result : result.concat(key)
+  }, [])
+}
```

Comment:
IMO, this would look a bit nicer with arrow functions:

```
export const getDirtyFields = (initialValues, values) =>
  _reduce(
    initialValues,
    (result, value, key) => _isEqual(value, vaules[key]) ? result : result.concat(key)
    []
  )